roles/heat/tasks/main.yml

```diff
@@ -49,14 +49,14 @@
     - heat_stack_user
   run_once: true
 
-- name: create heat stack admin
-  command: heat-keystone-setup-domain --stack-user-domain-name heat --stack-domain-admin heat_domain_admin --stack-domain-admin-password {{ secrets.stack_domain_admin_password }}
-  environment:
-      OS_USERNAME: provider_admin
-      OS_PASSWORD: "{{ secrets.provider_admin_password }}"
-      OS_AUTH_URL: "{{ endpoints.auth_uri }}"
-      OS_CACERT: /opt/stack/ssl/openstack.crt
-  run_once: true
+#- name: create heat stack admin
```

Comment:
yeah it failed and I ran it manually to get it to work and then had to comment out that those lines to get it to continue.